lib/double_linked_list.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
```

Comment:
Can you  please add the `/lib` to your `.gitignore`, it would make these PR's much more concise.